### ISSUE TITLE
Make a test for a build

### ISSUE THREAD

**Issue by lucienbill** [lucienbill/starterkit-selenium-js#9]
## Note
This issue can be separated into several issues.

## Version 0
In a real world, I like this usage:

> The test scenario that uses Selenium is integrated to a CI/CD pipeline.
> This pipeline builds the application in a ephemeral environment,
> then runs the units tests,
> then the Selenium tests.
> If and only if all the tests pass, then the build artefacts are deployed to a real environment

I would like to demo this kind of thing.

## Version 1
Bonus for the next version: let's imagine a CI/CD pipeline for the front-end part of the app.
My opinion is that our CI/CD test must only test the front-end of our app: the result of the tests should not depend upon the availability of the back-end.

In other words: I want to simulate the back-end, by using mocks.
It is doable with Selenium.

## Version 2
Bonus for the next version: when I mock an API call, this question comes up: _"what happens when the API evolves? How do I know it did, so I can update the mock?"_.
If the API has an Open API specification, things can be done, such as:
- automating the mock generation (if the spec is super mature)
- automatically detect that our test mocks an obsolete version of the API (much simpler to achieve)
